docker/1.0-1/final/Dockerfile_aarm64.cpu

```diff
@@ -0,0 +1,56 @@
+FROM sklearn-base:1.0-1-arm64-cpu-py3
```

Comment:
This is a separate feature from the sklearn version update right? Not fully sure about the usage of this, but will this be able to find the base image(and this final one) with the specified tag? The base docker file is suffixed as `aarm64` and not `arm64`.

Reply:
The `arm64` is related to graviton image, we can ignore it for now.

Reply:
I see, ok.